stable_baselines3/a2c/a2c.py

```diff
@@ -152,15 +154,15 @@ def train(self, gradient_steps: int, batch_size: Optional[int] = None) -> None:
     def learn(self,
               total_timesteps: int,
               callback: MaybeCallback = None,
-              log_interval: int = 100,
+              log_interval: int = 1,
```

Comment:
I know you did that for consistency but the thing is that A2C uses usually small `n_steps` and as a results, it floods the terminal (compared to PPO for instance)

Reply:
Oh wups, I am not sure how this variable got changed. I admit I did it for consistency, but I somehow missed the semantic meaning of this variable. I will update this to "100" again!